app/helpers/time_range_helper.rb

```diff
@@ -82,26 +82,21 @@ def to_s(format, which)
 
     def get_parts
       parts = {
-        :wday => Date::DAYNAMES[time.wday],
-        :month => Date::MONTHNAMES[time.month],
-        :day => time.day.to_s,
-        :year => time.year.to_s,
-        :at => "at" }
+        wday: time.strftime("%A"),
+        month: time.strftime("%B"),
+        day: time.day,
```

Comment:
The only thing that jumps out is `day:` and `year:` both being `Fixnum`s now instead of `Strings`. Is there a reason you removed the `#to_s`?